internal/db/schema/migrations/oss/postgres/42/01_create_kms_key_revocations.up.sql

```diff
@@ -0,0 +1,43 @@
+begin;
+
+    create table kms_key_revocation_status_enm (
+        string text primary key
+            constraint only_predefined_values_allowed
+            check(string in ('pending', 'running', 'completed', 'failed'))
+    );
+
+    comment on table kms_key_revocation_status_enm is
+        'Table holding valid statuses a key revocation can take';
+
+    insert into kms_key_revocation_status_enm (string)
+    values
+        ('pending'),
+        ('running'),
+        ('completed'),
+        ('failed');
+
+    create trigger 
+        mmutable_columns
+    before
+    update on kms_key_revocation_status_enm
+        for each row execute procedure immutable_columns('string');
+
+    create table kms_key_revocations(
```

Comment:
```suggestion
    create table kms_key_revocation(
```
Table names should be singular